spec/models/time_since_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe TimeSince, type: :model do
+  let(:time) { Time.zone.local(2021) }
```

Comment:
good tests 👍 

you may want to consider using Timecop to freeze time, rather than stubbing the Time class with allow().to receive, e.g.:

```
RSpec.describe TimeSince do
  around do |example|
    Timecop.freeze('2021-01-01 00:00:00')
    example.run
    Timecop.return
  end

 it 'should test something' do
   Time.local.now # this returns 01/01/2021 at 00:00:00
  ....
```

check the other rspecs for Timecop